.gitignore

```diff
@@ -620,3 +620,13 @@ MigrationBackup/
 # End of https://www.gitignore.io/api/osx,python,pycharm,windows,visualstudio,visualstudiocode
 opensourceleg/knee_encoder_map.npy
 opensourceleg/ankle_encoder_map.npy
+
+
+localtest.py
+opensourceleg/hardware/Re_Actuators.md
+opensourceleg/hardware/actuators_base.py
```

Comment:
Should this really be ignored?

Reply:
Those are @Robin0265's local files I think. He has an incoming PR with some of his changes.

Reply:
Ok cool, just checking so we stay tidy.

Reply:
Removed all of @Robin0265's local files.